data/human/free worlds 0 prologue.txt

```diff
@@ -1818,30 +2053,38 @@ mission "Rescue Katya 1"
 	
 	on offer
 		conversation
-			`You are on Bourne, the capital of the Free Worlds. Do you take them up on their job offer and volunteer to officially join them?`
+			`You are on Bourne, the capital of the Free Worlds. Do you take them up on their job offer and volunteer to officially join them? With tensions between the Free Worlds and Republic so high it's not unthinkable that they could go to war with one another, so it would be a good idea to have a warship and some spare funds before siding with the Free Worlds.`
 			choice
 				`	(Yes.)`
 				`	(Not yet.)`
 					defer
 			
+			branch mistrust
+				has "fw escort: mistrusted"
+			
 			`	You find the Free Worlds offices and give your name to the person at the front desk. A few minutes later Tomek Voigt, who manages supplies and fleet movements for the Council, comes down the stairs to greet you. "I'm so glad you've agreed to join us, Captain <last>," he says.`
-			`	That night you are introduced to the rest of the Council, and they take you out to dinner at a very fancy restaurant. Halfway through the third course of the meal, Tomek's communicator beeps. He reaches over to turn it off, then sees the message and turns pale. A hush falls over the table. "<system> has been captured by the Navy," he announces.`
+				goto next
+			
+			label mistrust
+			`	You find the Free Worlds offices and give your name to the person at the front desk. A few minutes later Tomek Voigt, who manages supplies and fleet movements for the Council, comes down the stairs to greet you. "Captain <last>," he says with a stern glare. "If it were up to me alone you would not be standing in front of me today, but it would appear that your continued help has earned you enough good will to surpass my mistrust of you.`
+			`	"But I'm a reasonable man," he continues, his expression changing to a smile. "If my friends trust you, then I will give you a second chance, even if it is against my better judgment. Now please, join us."`
+			
+			label next
+			`	That night you are introduced to the rest of the Council: Alondo Gruyere, the Council's chief diplomat, Freya Winters, an adept engineer and the Council's head of logistics, and Jean-Jacques "JJ" Soleau, an ex-Navy officer who now leads the training of the Free Worlds miltias. Katya Reynolds, the final member of the Council, is not present.`
```

Comment:
Freya does logistics, and Tomek does supplies? What's the difference?

Reply:
¯\\\_(ツ)\_/¯
MZ's work on that one, not mine.

Reply:
Supplies is making sure that there are people who can actually supply what you need, in the quantities you need them, on the timeline you want them - and if not, seeking other suppliers or investing in expanding the capacities of existing ones.

Logistics is corralling the products of your dozens to hundreds of suppliers, and potentially *their* suppliers if you're a government who's requisitioning vehicles and resources, and ensuring that all of those supplies actually end up where they're supposed to, when they're supposed to, and that suppliers and receivers both are _aware_ of when and where those supplies are ending up.

In a small project this might be the same person/department, but for anything large enough to require multiple people to manage it, these are absolutely separate jobs.